cdap-app-fabric/src/main/java/io/cdap/cdap/support/lib/SupportBundleOverallStatus.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.support.lib;
+
+import io.cdap.cdap.support.status.CollectionState;
+
+public class SupportBundleOverallStatus {
```

Comment:
nit: What is the difference between SupportBundleOverallStatus and SupportBundleStatus? Would calling one of them SupportBundleOperation or SupportBundleOperationStatus make more sense?

Reply:
yeah I will change SupportBundleOverallStatus to SupportBundleOperationStatus